www/base/src/app/home/home.tpl.jade

```diff
@@ -1,5 +1,5 @@
 .container
-  .row(ng-if="config.buildbotURL != baseurl")
+  .row(ng-if="config.buildbotURL != baseurl and config.buildbotURLAliases.indexof(baseurl) === -1")
```

Comment:
I think we can have the buildbotURLAliases include the buildbotURL so that this expression is simplified